client/src/interface/User.ts

```diff
@@ -1,6 +1,43 @@
 export interface User {
+  _id: string;
```

Comment:
Good work with the interfaces. Please also coordinate with the team so that the interfaces match the response data :)

Reply:
Will do that today.